internal/datastore/datastore.go

```diff
@@ -45,8 +45,8 @@ type Datastore interface {
 	GetPostsByDiscussionID(ctx context.Context, discussionID string) ([]*model.Post, error)
 	GetPostsByDiscussionIDIter(ctx context.Context, discussionID string) PostIter
 	GetPostContentByID(ctx context.Context, id string) (*model.PostContent, error)
-	PutPost(ctx context.Context, post model.Post) (*model.Post, error)
-	PutPostContent(ctx context.Context, postContent model.PostContent) error
+	PutPost(ctx context.Context, tx *sql2.Tx, post model.Post) (*model.Post, error)
```

Comment:
I feel like we're going to raise our eyebrows at the `sql2` in the future. Not sure what else we can do to fix it but just calling it out.

Reply:
once we swap out gorm, it'll take care of itself

Reply:
we could also just rename it to `pureSQL` or something